lib/assesments/assessment_2/fi_a2_bilal0x01/screens/profile/widgets/photos_section_bilal0x01.dart

```diff
@@ -0,0 +1,56 @@
+part of '../profile_screen_bilal0x01.dart';
+
+class _PhotosSection extends StatelessWidget {
+  const _PhotosSection({super.key});
+
+  Widget clippedImageBuilder(String imagePath) {
```

Comment:
Since this is being used at multiple places it's better to make a class, instead of a function, functions are pushed into stack everytime build is called, they affect performance, classes do not have this problem, you can define it locally outside the widget using _leading